.github/workflows/dart_package.yml

```diff
@@ -104,7 +104,7 @@ jobs:
       - name: 🧪 Run Tests
         run: |
           dart pub global activate coverage 1.2.0
-          dart test -j ${{inputs.concurrency}} --coverage=coverage --platform=${{inputs.platform}} && dart pub global run coverage:format_coverage --lcov ${{(inputs.check_ignore && '--check-ignore') || ''}} --in=coverage --out=coverage/lcov.info --packages=.dart_tool/package_config.json --report-on=${{inputs.report_on}} ${{(inputs.run_skipped && '--run-skipped') || ''}}
+          dart test -j ${{inputs.concurrency}} --coverage=coverage --platform=${{inputs.platform}} ${{(inputs.run_skipped && '--run-skipped') || ''}} && dart pub global run coverage:format_coverage --lcov ${{(inputs.check_ignore && '--check-ignore') || ''}} --in=coverage --out=coverage/lcov.info --packages=.dart_tool/package_config.json --report-on=${{inputs.report_on}}
```

Comment:
Am I missing something? Both seem to be equivalent?

Reply:
The current step misplaced the `--run-skipped` flag, applying it to the ` format_coverage` command, instead of the `dart test` command.

Reply:
You're right, thank you for catching this 💙 ! Seems that it sneaked in unintentionally within a community contribution.